.github/workflows/tests.yml

```diff
@@ -47,8 +47,28 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Download reference data
-        run: bash .ci-helpers/download_reference_data.sh
+      - name: Clone tardis-sn/tardis-refdata
+        uses: actions/checkout@v2
+        with:
+          repository: tardis-sn/tardis-refdata
+          path: tardis-refdata
+          lfs: false
+
+      - name: Create LFS file list
+        run: git lfs ls-files -l | cut -d' ' -f1 | sort > .lfs-assets-id
+        working-directory: tardis-refdata
+
+      - name: Restore LFS cache
+        uses: actions/cache@v2
+        id: lfs-cache
+        with:
+          path: tardis-refdata/.git/lfs
+          key: ${{ runner.os }}-lfs-${{ hashFiles('tardis-refdata/.lfs-assets-id') }}-v1
+
+      - name: Git LFS Pull
+        run: git lfs pull
+        working-directory: tardis-refdata
+        if: steps.lfs-cache.outputs.cache-hit != 'true'
```

Comment:
Apparently, the tests fail if we do not `git lfs pull`. Is it possible that we need to run `git lfs checkout` even if we recover the lfs stuff from the cache?

Reply:
Ok. This was the problem.

Reply:
Yes! Thank you for suggesting this. I did `git lfs checkout` in commit: https://github.com/tardis-sn/tardis/pull/2194/commits/8e68779e206ced452693cef26d5c22e64bcda896